src/Api/Health/HealthCheckServices.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Core.Settings;
+
+namespace Bit.Api.Health;
+
+internal static class HealthCheckServices
+{
+    public static void ConfigureHealthCheckServices(this IServiceCollection services,
+        GlobalSettings globalSettings, IHostEnvironment environment)
+    {
+        var builder = services.AddHealthChecks();
+        var identityUri = new Uri(globalSettings.BaseServiceUri.Identity + "/.well-known/openid-configuration");
+
+        if (!string.IsNullOrEmpty(GetConnectionString(globalSettings)))
+        {
+            //add custom db health check
+            builder.AddDatabaseCheck(globalSettings);
+        }
+
+        builder.AddUrlGroup(identityUri, "identity_server")
+            .AddRedis(globalSettings.Redis.ConnectionString)
+            .AddAzureQueueStorage(globalSettings.Storage.ConnectionString, name: "storage_queue")
+            .AddAzureQueueStorage(globalSettings.Events.ConnectionString, name: "events_queue")
+            .AddAzureQueueStorage(globalSettings.Notifications.ConnectionString, name: "notifications_queue")
+            .AddAzureServiceBusTopic(s => globalSettings.ServiceBus.ConnectionString,
+                s => globalSettings.ServiceBus.ApplicationCacheTopicName, name: "serviceBus")
+            .AddSendGrid(globalSettings.Mail.SendGridApiKey)
+            .AddCheck<AmazonSesHealthCheck>(nameof(AmazonSesHealthCheck));
+
+        //TODO: Decide if this is needed
+        //     .AddSmtpHealthCheck(setup =>
+        // {
+        //     setup.Host = globalSettings.Mail.Smtp.Host;
+        //     setup.Port = globalSettings.Mail.Smtp.Port;
+        //     setup.ConnectionType = SmtpConnectionType.PLAIN;
+        // }, "dev_mail_server");
```

Comment:
We can delete this, developers won't be using these health checks for the most part. When we tailor this more for self hosted and can add this back in.